module/epochs/qc_voter.go

```diff
@@ -92,14 +93,18 @@ func (voter *RootQCVoter) Vote(ctx context.Context, epoch protocol.Epoch) error
 
 		// for all attempts after the first, wait before re-trying
 		if attempts > 1 {
-			time.Sleep(voter.wait)
-		}
-
-		// check that our context is still valid
-		select {
-		case <-ctx.Done():
-			return fmt.Errorf("context cancelled: %w", ctx.Err())
-		default:
+			base := voter.wait << (attempts - 2)          // base wait period on an exponential backoff
+			jitter := float64(base) * rand.Float64() * .1 // add 10% jitter to avoid synchronization across cluster
+			wait := time.Duration(base) + time.Duration(jitter)
```

Comment:
Can we put this into a function like `getWaitTime(attemptNumber)` or something?

Reply:
Done in https://github.com/onflow/flow-go/pull/942/commits/f0fc6f38057e60a4765cf384bc5c6d15c9f58924